tools/nvqpp/nvq++.in

```diff
@@ -763,7 +763,7 @@ else # we honor the user's explicit command-line request.
 	fi
 fi
 if ${RUN_OPT}; then
-	OPT_PASSES=$(add_pass_to_pipeline "${OPT_PASSES}" "canonicalize,cse")
+	OPT_PASSES=$(add_pass_to_pipeline "${OPT_PASSES}" "distributed-device-call,canonicalize,cse")
```

Comment:
I think this ought to be under its own `-f` and `-fno-` flags so it can be more precisely selected/deselected.

(Defaulting to ON seems like the best here.)